dcos/package.py

```diff
@@ -1364,8 +1390,11 @@ def marathon_json(self, revision, options):
         :rtype: dict
         """
 
+        package_json = self.package_json(revision)
+        options = _merge_options(options, {"package": package_json})
```

Comment:
@tamarrow The user should not be able to overwrite stuff in `package.json`. That means that we need to merge user values with the default and then merge package.json with the previous result. Are we doing that? Can we have tests that check this?

Also, I thought we agreed to exit with an error if we overwrite package values.